Documentation/workflow/UsingYourBuild.md

```diff
@@ -44,22 +44,24 @@ dotnet new console
 
 ### Change project to be self-contained
 
-In order to update with your local changes the application needs to be self-contained, as opposed to running on the
+In order to update with your local changes, the application needs to be self-contained, as opposed to running on the
 shared framework.  In order to do that you will need to add a `RuntimeIdentifier` to your project.
 
-```
-  <PropertyGroup>
-    ...
-    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
-  </PropertyGroup>
+```xml
+<PropertyGroup>
+  ...
+  <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
Nit: I tend to prefer to just pass `-r win-64` to `dotnet publish`. It is faster/easier.